Make the ruff pre-commit hook fix issues when possible

### Proposal

<!-- Brief overview of the proposal -->

```
scripts/expire_accounts.py:1:1: I001 [*] Import block is un-sorted or un-formatted
  |
1 | / from datetime import datetime, UTC
2 | | import web
3 | | 
4 | | 
5 | | def delete_old_links():
  | |_^ I001
6 |       for doc in web.ctx.site.store.values(type="account-link"):
7 |           expiry_date = datetime.strptime(doc["expires_on"], "%Y-%m-%dT%H:%M:%S.%f")
  |
  = help: Organize imports

Found 1 error.
[*] 1 fixable with the `--fix` option.
```

> Can we add the --fix option to our import linting rule to fix some common errors?

~ Mek (in a Slack DM)

<!-- Designs / screenshots that may help reviewers understand your proposal -->


### Justification

<!-- Problem: What problem does this proposal address & for what audience(s)? -->

<!-- Impact: What's the predicted impact, how do we measure, & what does success look like? -->

<!-- Research: Are there any precedents or findings from other projects or studies that support this feature? (link or screenshot if possible) -->


### Breakdown


#### Requirements Checklist
<!-- What steps, functional pieces, or requirements need to be met to satisfy the proposal? -->
* [ ]

#### Related files
<!-- Files related to the implementation of feature; useful for good first issues -->
*

#### Stakeholders
<!-- @ tag stakeholders of this feature w/ questions or context as necessary -->
*

<hr>

#### Instructions for Contributors
<!-- Please leave the following reminder section as is to help new contributors and add instructions where necessary -->
Please [run these commands](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#working-on-your-branch) to ensure your repository is up to date **before** [creating a new branch](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#making-changes-and-creating-a-pull-request) to work on this issue and **each time after** pushing code to Github, because the pre-commit bot may add commits to your PRs upstream.

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -35,6 +35,7 @@ repos:
     rev: v0.6.7
     hooks:
       - id: ruff
+        args: [ --fix ]
 
   - repo: https://github.com/psf/black
     rev: 24.8.0

